modules/nw-egress-router-redirect-mode-ovn.adoc

```diff
@@ -3,11 +3,11 @@
 // * networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc
 
```

Comment:
@jboxman , I removed "pod" from the text because this work introduces a dedicated custom resource.  If removing pod was a mistake, just let me know and thanks.

Reply:
++, also, this is now deployed using a 'deployment' rather than a specific pod resource, just in case you'd like to mention it.